vowpalwabbit/config.h.in

```diff
@@ -0,0 +1 @@
+#define PACKAGE_VERSION "${PACKAGE_VERSION}"
```

Comment:
>PACKAGE_VERSION [](start = 27, length = 15)

Is this set by CMake now?

Reply:
yes, through configure_file

---
In reply to: [192517398](https://github.com/JohnLangford/vowpal_wabbit/pull/1492#discussion_r192517398) [](ancestors = 192517398)